### PR TITLE
Add isTrue and isFalse predicates

### LINKED ISSUE #255: RA.isTrue, RA.isFalse
Don't forget to refactor code that now uses constructs like
```javascript
equals(true)
equals(false)
```

### PR DESCRIPTION
**Pull request by Undistraction**
Adds predicates for testing the Boolean primatives `true` and `false`.

Note: I have taken the decision not to try and support values created using the Boolean function as a constructor - `new Boolean(false)`, as the values returned are not Boolean primitives but are actually objects. I think this is such an edge-case and is potentially such a deep rabbit hole that it isn't worth the added complexity. Very much open to discussion on this though.

Closes #255

### CODE REVIEW

File: src/lensSatisfies.js [hunk 1]
```diff
@@ -19,13 +21,13 @@ import { view, curryN, equals, pipe } from 'ramda';
...
  * @example
  *
- * RA.lensSatisfies(R.equals(true), R.lensIndex(0), [false, true, 1]); // => false
- * RA.lensSatisfies(R.equals(true), R.lensIndex(1), [false, true, 1]); // => true
- * RA.lensSatisfies(R.equals(true), R.lensIndex(2), [false, true, 1]); // => false
- * RA.lensSatisfies(R.identity, R.lensProp('x'), { x: 1 }); // => false
+ * RA.lensSatisfies(RA.isTrue, R.lensIndex(0), [false, true, 1]); // => false
+ * RA.lensSatisfies(RA.isTrue, R.lensIndex(1), [false, true, 1]); // => true
+ * RA.lensSatisfies(RA.isTrue, R.lensIndex(2), [false, true, 1]); // => false
+ * RA.lensSatisfies(RA.identity, R.lensProp('x'), { x: 1 }); // => false
```
Comment (guillaumearm) [inline src/lensSatisfies.js]: R.identity

**Comment by Undistraction** [comment]
@guillaumearm Thanks. I'll wait for @char0n the human linter ;) to have a look before merging.

File: src/index.d.ts [hunk 2]
```diff
@@ -784,6 +784,18 @@ declare namespace RamdaAdjunct {
          */
         isPromise(val: any): val is Promise<any>;
 
+        /**
+         * Checks if input value is the Boolean primative `true`. Will return false for all values
```
Comment (char0n) [inline src/index.d.ts]: primative -> primitive

```
Will return false for Boolean objects created using the `Boolean` function as a constructor

File: src/index.d.ts [hunk 3]
```diff
@@ -784,6 +784,18 @@ declare namespace RamdaAdjunct {
...
+         * created using the `Boolean` function as a constructor.
+         */
+        isTrue(val: any): boolean;
+
+        /**
+         * Checks if input value is the Boolean primative `false`. Will return false for all values
```
Comment (char0n) [inline src/index.d.ts]: primative -> primitive
```
Will return false for Boolean objects created using the `Boolean` function as a constructor
```

File: src/isFalse.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import { equals } from 'ramda';
+
+/**
+ * Checks if input value is the Boolean primative `false`. Will return false for all values created
```
Comment (char0n) [inline src/isFalse.js]: primative -> primitive 

```
Will return false for Boolean objects created using the `Boolean` function as a constructor
```

File: src/isFalse.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+ * using the `Boolean` function as a constructor.
+ *
+ * @func isFalse
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.6.0|v2.6.0}
+ * @category Type
+ * @sig * -> Boolean
+ * @param {*} val The value to test
+ * @return {boolean}
+ * @see {@link https://developer.mozilla.org/en-US/docs/Glossary/Truthy|truthy}, {@link RA.isTrue|isTrue}, {@link RA.isTruthy|isTruthy}, {@link RA.isFalsy|isFalsy}
```
Comment (char0n) [inline src/isFalse.js]: Pls drop `{@link https://developer.mozilla.org/en-US/docs/Glossary/Truthy|truthy}`

It is part of `RA.isTruthy`

File: src/isTrue.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
+import { equals } from 'ramda';
+
+/**
+ * Checks if input value is the Boolean primative `true`. Will return false for all values created
```
Comment (char0n) [inline src/isTrue.js]: primative -> primitive
```
Will return false for Boolean objects created using the `Boolean` function as a constructor.
```

File: src/isTrue.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+ * using the `Boolean` function as a constructor.
+ *
+ * @func isTrue
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.6.0|v2.6.0}
+ * @category Type
+ * @sig * -> Boolean
+ * @param {*} val The value to test
+ * @return {boolean}
+ * @see {@link https://developer.mozilla.org/en-US/docs/Glossary/Truthy|truthy}, {@link RA.isFalse|isFalse}, {@link RA.isTruthy|isTruthy}, {@link RA.isFalsy|isFalsy}
```
Comment (char0n) [inline src/isTrue.js]: Drop `{@link https://developer.mozilla.org/en-US/docs/Glossary/Truthy|truthy}`

This is already linked through `RA.isTruthy`

File: test/isFalse.js [hunk 8]
```diff
@@ -0,0 +1,40 @@
+import * as RA from '../src/index';
+import eq from './shared/eq';
+import Symbol from './shared/Symbol';
+import args from './shared/arguments';
+
+describe('isFalse', function() {
+  it("tests whether a value is 'true'", function() {
```
Comment (char0n) [inline test/isFalse.js]: 'true' -> 'false'

"tests whether a value is 'false' primitive"

File: test/isTrue.js [hunk 9]
```diff
@@ -0,0 +1,40 @@
+import * as RA from '../src/index';
+import eq from './shared/eq';
+import Symbol from './shared/Symbol';
+import args from './shared/arguments';
+
+describe('isTrue', function() {
+  it("tests whether a value is 'true'", function() {
```
Comment (char0n) [inline test/isTrue.js]: "tests whether a value is 'true' primitive"

**Comment by Undistraction** [comment]
@char0n Should be ready for a merge.

File: src/index.d.ts [hunk 10]
```diff
@@ -784,6 +784,17 @@ declare namespace RamdaAdjunct {
          */
         isPromise(val: any): val is Promise<any>;
 
+        /**
+         * Checks if input value is the Boolean primitive `true`.Will return false for Boolean
```
Comment (char0n) [inline src/index.d.ts]: .Will -> . Will

**Comment by char0n** [comment]
Well done. Now have a 2.6.0 release

**Pull request merged**